DOCS-2055: update bancontact postepay visa vpay testing details (#1805)
MultiSafepay/docs: content/payments/methods/banks/bank-transfer/integration-and-testing.md

---

**Note:** 

> - You can't test the following:

Reviewer: ```suggestion
- You can't test:
```

>   - Sending a refund API request

Reviewer: ```suggestion
  - Refunding via the API
```

>   - Sending a direct API request with an IBAN to test different transaction statuses

Reviewer: ```suggestion
  - Sending direct API requests with an IBAN to test different transaction statuses
```
